src/upload-button.tsx

```diff
@@ -33,7 +33,7 @@ import { Flex, FlexItem } from "@patternfly/react-core/dist/esm/layouts/Flex";
 import { TrashIcon } from "@patternfly/react-icons";
 
 import cockpit, { BasicError } from "cockpit";
-import type { FileInfo } from "cockpit/fsinfo";
+import { fsinfo, type FileInfo } from "cockpit/fsinfo.ts";
```

Comment:
Do we need `.ts`? If not we should remove it

Reply:
Technically not, but it upsets our linter for some reason, thinking about it isn't the import invalid JS/TS syntax anyway?

Reply:
Weird. I do think it is slightly invalid to use extensions for js/ts files but don't quote me on that

Reply:
This is a ridiculously complicated topic.  For now my advice is "don't rock the boat".

Currently, esbuild sort protects us from having to care about this, which is why anything goes.  We'll probably change our approach at some point...